tests/test_deploy_reqs.py

```diff
@@ -6,23 +6,42 @@
 import unittest
 import os
 import tempfile
-from mock import Mock
+from mock import Mock, patch
+from click.testing import CliRunner
 
 from shub import deploy_reqs
 
 
+@patch('shub.deploy_reqs.utils.build_and_deploy_egg')
 class TestDeployReqs(unittest.TestCase):
-    def test_can_decompress_downloaded_packages_and_call_deploy_reqs(self):
+    VALID_APIKEY = '1234'
+
+    def setUp(self):
+        self.runner = CliRunner()
+        os.environ['SHUB_APIKEY'] = self.VALID_APIKEY
```

Comment:
@eliasdorneles ok, CliRunner also allows us to override environment variables in an isolated way. Using it now. I'll refactor some tests after these PRs are merged to DRY them a little bit. This is repeated in other tests also.